suite-native/transactions/src/components/TransactionsEmptyState.tsx

```diff
@@ -36,8 +36,8 @@ export const TransactionsEmptyState = ({ accountKey }: { accountKey: string }) =
     const { applyStyle } = useNativeStyles();
 
     const handleReceive = () => {
-        navigation.navigate(AppTabsRoutes.SendReceiveStack, {
-            screen: SendReceiveStackRoutes.Receive,
+        navigation.navigate(AppTabsRoutes.ReceiveStack, {
+            screen: ReceiveStackRoutes.Receive,
```

Comment:
Should not this go to ReceiveModal? Not sure.

Reply:
https://github.com/trezor/trezor-suite/pull/8316/commits/eaebbfb6a11e30414886e378d6cc6708f05b6ca5